foundation/application.go

```diff
@@ -97,6 +103,26 @@ func (app *Application) Publishes(packageName string, paths map[string]string, g
 	}
 }
 
+func (app *Application) Version() string {
+	return support.Version
+}
+
+func (app *Application) GetLocale(ctx context.Context) string {
+	return app.MakeLang(ctx).GetLocale()
+}
+
+func (app *Application) SetLocaleByHttp(ctx http.Context, locale string) {
+	app.MakeLang(ctx).SetLocale(locale)
```

Comment:
```suggestion
	ctxWithLocale := app.MakeLang(ctx).SetLocale(locale)
```

We need to set `ctxWithLocale` into `http.Context`, otherwise, the `locale` will be lost. In order to achieve this purpose, we need to add a new `SetContext` method to `contracts/http/context.go::Context`. We can add it in another PR and need to modify goravel/gin and goravel/fiber, I can do it if you want.

Reply:
Okay i will add SetContext method in Context intereface.